src/main/java/org/alfresco/rest/api/search/SearchSQLApiWebscript.java

```diff
@@ -80,6 +82,10 @@ public void execute(WebScriptRequest webScriptRequest, WebScriptResponse res) th
             //Turn JSON into a Java object representation
             SearchSQLQuery searchQuery = extractJsonContent(webScriptRequest, assistant.getJsonHelper(), SearchSQLQuery.class);
             SearchParameters sparams = buildSearchParameters(searchQuery);
+            if(searchQuery.getFormat().equalsIgnoreCase("solr"))
```

Comment:
Is it possible that getFormat returns null? I prefer to do the check the other way round to avoid a NPE, i.e.

"solr".equalsIgnoreCase(searchQuery.getFormat())

but this is a personal preference.